test-infra/prow/config.yaml

```diff
@@ -119,6 +119,22 @@ periodics:
     - name: compute-image-tools-test-service-account
       secret:
        secretName: compute-image-tools-test-service-account
+ - name: osconfig-tests
+   interval: 3h
+   agent: kubernetes
+   spec:
+    containers:
+    - image: gcr.io/compute-image-tools-test/osconfig-tests:latest
+      args:
+      - "-out_path=/artifacts/junit.xml"
```

Comment:
Do these tests share the project pool with the rest of the tests in this repo?

Reply:
For now they do (right now there are no locking concerns), I plan on making new projects for osconfig tests soon